### PR TITLE
Add anime topic

### PR DESCRIPTION
**Pull request by Nekidev**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> Anime has become really popular over the years. There are many projects with this topic, so some information should be added.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [x] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Added the `anime` topic. This topic's description was taken from [Wikipedia](https://en.wikipedia.org/wiki/Anime).

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by SatoshiNakamoto-MAM** [comment]
Sorry im the creater of blockchain and bitcoin i am satoshi nakamoto

**Review by kenyonj — CHANGES_REQUESTED** [review]
Please address the CI failures and we can review. Thanks!

File: topics/anime/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: animes
+display_name: Anime
+short_description: Anime is hand-drawn and computer-generated animation originating from Japan.
+topic: anime
+wikipedia_url: https://en.wikipedia.org/wiki/Anime
+---
```
Comment (Kristories) [inline topics/anime/index.md:7]: Logo key should be `anime.png`, but was missing

**Review by Kristories — COMMENTED** [review]
Images should not exceed 75,000 bytes, got 150,897 bytes

**Comment by ChristopherLeeFalck** [comment]
> 
> 
> ### Please confirm this pull request meets the following requirements:
> 
> 
> 
> - [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
> 
> - [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).
> 
> 
> 
> ### Which change are you proposing?
> 
> 
> 
>   - [ ] Suggesting edits to an existing topic or collection
> 
>   - [x] Curating a new topic or collection
> 
>   - [ ] Something that does not neatly fit into the binary options above
> 
> 
> 
> ---
> 
> 
> 
> 
> 
> ### Editing an existing topic or collection
> 
> 
> 
> I'm suggesting these edits to an existing topic or collection:
> 
> - [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
> 
> - [ ] Content (and my changes are in `index.md`)
> 
> 
> 
> > Please replace this line with an explanation of why you think these changes should be made.
> 
> 
> 
> 
> 
> ### Curating a new topic or collection
> 
> 
> 
> - [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
> 
> - [x] My folder contains a `*.png` image (if applicable) and `index.md`
> 
> - [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>
> 
> 
> 
> > Anime has become really popular over the years. There are many projects with this topic, so some information should be added.
> 
> 
> 
> 
> 
> ### Something that does not neatly fit into the binary options above
> 
> 
> 
> - [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
> 
> - [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
> 
> - [x] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs
> 
> 
> 
> > Added the `anime` topic. This topic's description was taken from [Wikipedia](https://en.wikipedia.org/wiki/Anime).
> 
> 
> 
> ---
> 
> 
> 
> **Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**
> 
>

**Comment by IcebergVerse** [comment]
bundle install
bundle exec rubocop

**Pull request closed without merging**